swagger.json

```diff
@@ -6,13 +6,67 @@
     "description": "Barefoot Nomad - Making company travel and accomodation easy and convinient.",
     "contact": {}
   },
+  "host": "",
```

Comment:
Great job bro

Reply:
Thanks, I appreciate you taking time out to do this